pkg/client/util_test.go

```diff
@@ -391,10 +391,12 @@ func TestReconcilePostgres(t *testing.T) {
 				modifyFunc:       nil,
 			},
 			want: &v1alpha1.Postgres{
-				TypeMeta: metav1.TypeMeta{
-					Kind:       "Postgres",
-					APIVersion: "integreatly.org/v1alpha1",
-				},
+				// Commented in MGDAPI-6290, as to have these fields polulated the Object should be runtime.Unstructured
```

Comment:
if this block of code is not used, please remove.

Reply:
done, removed. Thank you